tests/Auto/Animator/ImageCreator_test.py

```diff
@@ -26,11 +25,7 @@ def test_imagecreator():
     allocation_in_frame = AllocationInFrame()
     allocation_in_frame.words = {"word": (30, (50, 50))}  # dictionary
     position_in_frames.add("2023_04_01", allocation_in_frame)
-    create_images(
-        position_in_frames,
-        image_size=(100, 100),
-        font_path=DEFAULT_ENG_FONT_PATH,
-    )
+    create_images(position_in_frames, Config())
     test_path = os.path.join(DEFAULT_OUTPUT_PATH, "*.png")
```

Comment:
Although I wrote this line, change this line to """os.path.join(DEFAULT_OUTPUT_PATH, "2023_04_01.png")"""